### ISSUE TITLE
Running FreqTrade on VPS with 1 core

### ISSUE THREAD

**Issue by Jay-ArBrouillard** [freqtrade/freqtrade#4680]
## Your question

Is Hyperopt the only part of FreqTrade that requires 2 cores? I’m running on VPS and only using trade module and nothing else. I’m wondering if anyone can confirm it will run fine using 1 core instead of 2 to save $10/month?

**Comment by xmatthias** [comment]
It will heavily depend on your strategy, but i would not see using 1 core safe (not for anything, really).
It can run fine 90% of the time - but when you really need it to work, it'll fail you due to some odd coincidence.

I do however highly discourage using 1 core as that can cause very strange behaviour which you will not be able to fully understand in the end.

As only 1 core is available, EVERYTHING that runs on the box must share this core - so a ssh login / automatic system upgrade check can lock the only available core for some time, causing freqtrade to be delayed.

To summarize - it's clearly not supported, and should you run into errors, then be sure verify that this issue occurs also on a supported system configuration.

**Comment by rylos** [comment]
I use single cpu (vCPU) amazon Graviton C6G istance with up-to-date freqtrade, medium complex strategy (lot of indicators using pandas ta lib). No other tasks in background. Working like a charm (medium cpu usage  low).
Regards

**Comment by Jay-ArBrouillard** [comment]
Thanks everyone I will keep this in mind.
